dist/bin/kctf-cluster

```diff
@@ -125,6 +125,13 @@ function kctf_cluster_start {
 }
 
 function kctf_cluster_stop {
+  echo "deleting all challenges so that load balancers etc can be cleaned up"
```

Comment:
can we delete the default namespace?

Reply:
I don't think so.
That reminds me, this will fail if someone runs challenges in a different namespace. We should probably search for challenges in all namespaces.

Reply:
ah, yea